.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Request] "
+labels: enhancement
+assignees: ''
+
+---
+
+**Request**
+<!-- Here you will write what you want to suggest -->
+
+**Explain it to me like I was five**
```

Comment:
Reword 
`+ **Explain it to me like I was five**`
to
`+ **Explain it to me like I'm five**`